src/errors/VersionMismatchError.ts

```diff
@@ -31,7 +31,6 @@ function tmpl(strings: TemplateStringsArray, requiresUpdate: string): string {
 export default class VersionMismatchError extends Error {
   public requiresUpdate: RequiresUpdate;
   public code: ErrorCode;
-  public versions?: Versions;
```

Comment:
This is unused in the constructor. Should this be removed or is it used somewhere?

Reply:
Probably a refactoring mistake. If you remove it and it compiles without errors, then it's a mistake.